packages/vats/src/vat-zoe.js

```diff
@@ -11,7 +12,41 @@ export function buildRootObject(vatPowers, vatParameters) {
         feeIssuerConfig,
         vatParameters.zcfBundleName,
       );
+
+      const zoeWithoutInstallation = Far('zoe without install/startInstance', {
+        ...zoeService,
+        install: (..._args) => assert.fail('contract installation prohibited'),
```

Comment:
This indicates to me that they just be removed from the ZoeService.

Reply:
Then how does any privileged component install a contract?

Reply:
oh... I see your comment above...

I consider Zoe a "no user serviceable parts inside" component. So that's outside the scope of this PR.